### PR TITLE
doc: update the rados namespace docs

### PR DESCRIPTION
**Pull request by theanalyst**
Since last update was as of firefly and we do use namespaces in RGW

Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

### CODE REVIEW

**Comment by theanalyst** [comment]
jenkins test this please

File: doc/rados/operations/user-management.rst [hunk 1]
```diff
@@ -218,10 +218,10 @@ namespace such that reads and writes by the user take place only within the
...
 by users who have access to the namespace.
 
-.. note:: Currently (i.e., ``firefly``), namespaces are only useful for 
-   applications written on top of ``librados``. Ceph clients such as block 
-   device, object storage and file system do not currently support this 
-   feature.
+.. note:: namespaces are primarily useful for applications written on top of
+   ``librados`` where the logical grouping can alleviate the need to create
+   different pools. Ceph radosgw (from ``luminous``) uses namespaces for various
+   metadata objects.
```
Comment (cbodley) [inline doc/rados/operations/user-management.rst]: capitalize `Namespaces` at the beginning of the sentence, and `Ceph radosgw` -> `Ceph Object Gateway`

Comment (theanalyst) [inline doc/rados/operations/user-management.rst]: ack

**Comment by theanalyst** [comment]
@cbodley updated, sorry for letting this slide

**Comment by cbodley** [comment]
jenkins test this please

**Comment by theanalyst** [comment]
@cbodley Ok to merge?

**Pull request merged**